<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主窗口</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>981</width>
      <height>731</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Welcome</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>701</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>压水堆核电厂二回路热力系统初步设计(V0.1.0)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>620</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Designed by euaurora</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_newPro">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>290</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>新建项目</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>231</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>哈尔滨工程大学核科学与技术学院</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_exit">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>470</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>退出程序</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>80</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>核工程与核技术专业本科生课程设计（二）</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_openPro">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>380</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>打开项目</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件(&amp;F))</string>
    </property>
    <widget class="QMenu" name="menu_save">
     <property name="title">
      <string>另存为</string>
     </property>
     <addaction name="action_saveFulu"/>
     <addaction name="action_saveCode"/>
     <addaction name="action_savePlot"/>
    </widget>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="action_openFile"/>
    <addaction name="menu_save"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_plot">
    <property name="title">
     <string>绘制(&amp;P)</string>
    </property>
    <addaction name="action_structure"/>
    <addaction name="action_plot"/>
   </widget>
   <widget class="QMenu" name="menu_par">
    <property name="title">
     <string>参数(&amp;V)</string>
    </property>
    <addaction name="action_initial"/>
    <addaction name="action_showfulu"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_par"/>
   <addaction name="menu_plot"/>
  </widget>
  <action name="action_new">
   <property name="text">
    <string>新建项目</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>打开项目</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出程序</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="action_structure">
   <property name="text">
    <string>原理流程图</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
  <action name="action_initial">
   <property name="text">
    <string>输入原始参数</string>
   </property>
   <property name="shortcut">
    <string>Alt+V</string>
   </property>
  </action>
  <action name="action_saveFulu">
   <property name="text">
    <string>附录数据</string>
   </property>
  </action>
  <action name="action_saveCode">
   <property name="text">
    <string>Python代码</string>
   </property>
  </action>
  <action name="action_savePlot">
   <property name="text">
    <string>热线图</string>
   </property>
  </action>
  <action name="action_plot">
   <property name="text">
    <string>查看热线图</string>
   </property>
  </action>
  <action name="action_showfulu">
   <property name="text">
    <string>查看附录数据</string>
   </property>
  </action>
  <action name="action_openFile">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
